web_src/js/components/RepoActionView.vue

```diff
@@ -20,7 +20,7 @@
               <SvgIcon name="octicon-meter" class="ui text yellow" class-name="job-status-rotate" v-else-if="job.status === 'running'"/>
               <SvgIcon name="octicon-x-circle-fill" class="red" v-else/>
               {{ job.name }}
-              <button class="job-brief-rerun" @click="rerunJob(index)" v-if="job.canRerun">
+              <button class="job-brief-rerun" @click.prevent="rerunJob(index)" v-if="job.canRerun">
```

Comment:
If there is no `form`,   then `prevent` is not necessary IMO

Reply:
> If there is no `form`, then `prevent` is not necessary IMO

No, the `prevent` is not for `form`, it's for `a`. It will break again if I remove it.

Reply:
Oh, I see there is a `<a>` in its parents, so the `prevent` is required.